src/main/java/org/mitre/synthea/world/agents/PayerManager.java

```diff
@@ -0,0 +1,388 @@
+package org.mitre.synthea.world.agents;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.mitre.synthea.helpers.Config;
+import org.mitre.synthea.helpers.SimpleCSV;
+import org.mitre.synthea.helpers.Utilities;
+import org.mitre.synthea.world.agents.behaviors.payeradjustment.IPayerAdjustment;
+import org.mitre.synthea.world.agents.behaviors.payeradjustment.PayerAdjustmentFixed;
+import org.mitre.synthea.world.agents.behaviors.payeradjustment.PayerAdjustmentNone;
+import org.mitre.synthea.world.agents.behaviors.payeradjustment.PayerAdjustmentRandom;
+import org.mitre.synthea.world.agents.behaviors.planeligibility.PlanEligibilityFinder;
+import org.mitre.synthea.world.agents.behaviors.planfinder.IPlanFinder;
+import org.mitre.synthea.world.agents.behaviors.planfinder.PlanFinderBestRates;
+import org.mitre.synthea.world.agents.behaviors.planfinder.PlanFinderGovPriority;
+import org.mitre.synthea.world.agents.behaviors.planfinder.PlanFinderRandom;
+import org.mitre.synthea.world.concepts.HealthRecord.EncounterType;
+import org.mitre.synthea.world.concepts.healthinsurance.InsurancePlan;
+import org.mitre.synthea.world.geography.Location;
+
+/**
+ * A class that maintains and manages Payers.
+ */
+public class PayerManager {
+
+  // Payer adjustment algorithm choices:
+  private static final String NONE = "none";
+  private static final String FIXED = "fixed";
+
+  public static final String GOV_OWNERSHIP = "GOVERNMENT";
+  public static final String PRIVATE_OWNERSHIP = "PRIVATE";
+  public static final String NO_INSURANCE = "NO_INSURANCE";
+
+  public static final String MEDICARE =
+      Config.get("generate.payers.insurance_companies.medicare", "Medicare");
+  public static final String MEDICAID =
+      Config.get("generate.payers.insurance_companies.medicaid", "Medicaid");
+  public static final String DUAL_ELIGIBLE =
+      Config.get("generate.payers.insurance_companies.dual_eligible", "Dual Eligible");
+
+  /* ArrayList of all Private Payers imported. */
+  private static List<Payer> privatePayers = new ArrayList<Payer>();
+  /* Map of all Government Payers imported. */
+  private static Map<String, Payer> governmentPayers = new HashMap<String, Payer>();
+  /* No Insurance Payer. */
+  public static Payer noInsurance;
+
+  /* U.S. States loaded. */
+  private static Set<String> statesLoaded = new HashSet<String>();
+
+  /* Payer Finder. */
+  private static IPlanFinder planFinder;
+  // Payer selection algorithm choices:
+  private static final String RANDOM = "random";
+  private static final String BESTRATE = "best_rate";
```

Comment:
This is really nit-picky, but suggest that you are consistent in constant naming. The rest of the file uses underscores to replace spaces as opposed to omitting them.